Library/Homebrew/utils/analytics.sh

```diff
@@ -14,17 +22,15 @@ setup-analytics() {
   if [[ "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsmessage)" != "true" ||
         "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsdisabled)" = "true" ]]
   then
```

Comment:
That makes sense. And the user can manually remove it if they really do want to reset their ID for some reason.